### ISSUE TITLE
Improve the ability to debug sentry integration

### ISSUE THREAD

**Issue by psibi** [getsentry/sentry-rust#620]
I have been trying to integrate sentry tracing with to one of the applications at work and I'm having some issues with it.

When trying to debug it further, I provide these options:

```
            sentry::ClientOptions {
                release: sentry::release_name!(),
                session_mode: sentry::SessionMode::Application,
                debug: true,
                // Have 1% sampling rate at production
                traces_sample_rate: 1.0,
                ..Default::default()
            },
```

And when the debug is `true`, it provides output like this:

```
[sentry] Get response: `{"id":"ae0962de93bb4e4bbafa958fb4737a44"}`
```

Unfortunately it's not very helpful for my case. Is there a way for me to understand what that id is and how to correlate with it ? Probably there is a documentation related to this, but I'm not able to find it in the crate docs or just general searching.

**Comment by Swatinem** [comment]
I agree that the response itself might not be as useful. At this point the transport is dealing with `Envelope`s and it might be possible to at least debug-print the types of envelope items.
